Content.Server/Power/Components/BatterySelfRechargerComponent.cs

```diff
@@ -9,5 +9,11 @@ public sealed partial class BatterySelfRechargerComponent : Component
         [ViewVariables(VVAccess.ReadWrite)] [DataField("autoRecharge")] public bool AutoRecharge { get; set; }
 
         [ViewVariables(VVAccess.ReadWrite)] [DataField("autoRechargeRate")] public float AutoRechargeRate { get; set; }
+
+        [ViewVariables(VVAccess.ReadWrite)] [DataField("autoRechargePause")] public bool AutoRechargePause { get; set; } = false;
+
+        [ViewVariables(VVAccess.ReadWrite)] [DataField("autoRechargePauseTime")] public float AutoRechargePauseTime { get; set; } = 0f;
+
+        [ViewVariables(VVAccess.ReadWrite)] [DataField("autoRechargeCooldown")] public float AutoRechargeCooldown { get; set; } = 0f;
```

Comment:
```suggestion
        [DataField]
        public bool AutoRecharge { get; set; }

        [DataField]
        public float AutoRechargeRate { get; set; }

        [DataField]
        public bool AutoRechargePause { get; set; } = false;

        [DataField]
        public float AutoRechargePauseTime { get; set; } = 0f;

        [DataField]
        public float AutoRechargeCooldown { get; set; } = 0f;
```
Yeah, `ViewVariables(VVAccess.ReadWrite)` is included in DataField. The names are automatically converted to camelCase as well. And give the datafields summary strings for documentation what they do.
(I did not check the rest of the code)